app/middleware/get_gateway_account.js

```diff
@@ -25,7 +26,9 @@ module.exports = function (req, res, next) {
   }
   return connectorClient.getAccount(params)
     .then(data => {
-      const SUPPORTS_3DS = (EPDQ_3DS_ENABLED === 'true') ? ['worldpay', 'epdq'] : ['worldpay']
+      // env var values are treated as text so the comparisaon is done for text
+      let SUPPORTS_3DS = (EPDQ_3DS_ENABLED === 'true') ? ['worldpay', 'epdq'] : ['worldpay']
+      if (SMARTPAY_3DS_ENABLED === 'true') { SUPPORTS_3DS = _.concat(SUPPORTS_3DS, ['smartpay']) }
```

Comment:
This bit is okay, but at a glance bit cryptic to as the same thing handled slightly differently. How about something  like
```
const SUPPORTS_3DS = ['worldpay']
if(EPDQ_3DS_ENABLED === 'true') {
    _.concat(SUPPORTS_3DS, ['epdq'])
}
if(SMARTPAY_3DS_ENABLED === 'true') {
    _.concat(SUPPORTS_3DS, ['smartpay'])
}
```

Reply:
I was thinking to use that style as well, I just pick one